client/app/scripts/liveblog-edit/blog.service.js

```diff
@@ -30,7 +30,7 @@ export default function blogService(api, $q, $rootScope, config) {
     * @param {object} blog
     * @return {promise} public_url
     **/
-    function getPublicUrl(blog) {
+    function getPublicUrl(blog, $scope = {}) {
```

Comment:
I think passing the `$scope` as parameter this way is not a good practice. Can you please explain the reason of this? Can we improve it? Thanks

Reply:
The reason was to update the pulicUrl attribute in the scope when the publicUrl arrives. Okay, I will improve it.